docs/customization.rst

```diff
@@ -119,174 +117,156 @@ options are:
         'globaltoc_depth': 2
     }
 
-Heroes
-==========
+.. confval:: hero
 
-To set the hero's text for an individual page, use the ``:hero:`` directive for the desired page.
-If not specified, then the page will not have a hero section.
+    To set the hero's text for an individual page, use the ``:hero:`` directive for the desired page.
```

Comment:
I think this is called a document metadata field:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadata

Reply:
```suggestion
    To set the hero's text for an individual page, use the ``:hero:`` metadata field for the desired page.
```